pages/blog/posts/schema-static-analysis.md

```diff
@@ -135,7 +135,7 @@ Dynamic references, on the other hand, generally boil down to one thing: dynamic
 
 ### Root schema dynamics
 
-Last year I wrote a [post](./dynamicref-and-generics) describing how to use `$dynamicRef` to model generic types in languages.  The idea went like this:
+Last year I wrote a [post](/blog/posts/dynamicref-and-generics) describing how to use `$dynamicRef` to model generic types in languages.  The idea went like this:
```

Comment:
Why not (dynamicref-and-generics) instead of (/blog/posts/dynamicref-and-generics)?

Reply:
Thanks for the positive feedback! 

Here's a quick overview of the link approach:

- At the time of this PR, the link checker we expected to use couldn't resolve paths in the repo context. As a workaround, I adopted the "/blog/posts/" convention.
- Currently, we're in the process of setting up a more robust link checker that will operate within the PR file context. 
- Both approaches have their merits, and we're flexible – either sticking with the current convention or opting for cleaner relative paths.

Let me know your preference.

Reply:
The url here and the rest of the links from one blog to another should be just the name of the page because all are in the same folder.